Content.Server/Storage/Components/ServerStorageComponent.cs

```diff
@@ -629,5 +632,39 @@ private void PlaySoundCollection()
         {
             SoundSystem.Play(Filter.Pvs(Owner), StorageSoundCollection.GetSound(), Owner, AudioParams.Default);
         }
+
+        [Verb]
+        private sealed class ToggleOpenVerb : Verb<ServerStorageComponent>
+        {
+            public override bool AlternativeInteraction => true;
+
+            protected override void GetData(IEntity user, ServerStorageComponent component, VerbData data)
+            {
+                if (!EntitySystem.Get<ActionBlockerSystem>().CanInteract(user))
+                {
+                    data.Visibility = VerbVisibility.Invisible;
+                    return;
+                }
+
+                // Does this player currently have the storage UI open?
+                var session = user.GetComponent<ActorComponent>().PlayerSession;
```

Comment:
`Get` instead of `TryGet` is suss but I guess it's a safe assumption that anything opening a storage UI is an actor

Reply:
Still better safe than sorry. Changed to use Get...OrNull, and now hide the verb if null.